examples/dynamic-lora-sidecar/sidecar/sidecar.py

```diff
@@ -0,0 +1,204 @@
+import requests
+import yaml
+import time
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+import logging
+import datetime
+import os
+
+CONFIG_MAP_FILE = os.environ.get("DYNAMIC_LORA_ROLLOUT_CONFIG", "configmap.yaml")
+DYNAMIC_LORA_FLAG = "VLLM_ALLOW_RUNTIME_LORA_UPDATING"
+BASE_FIELD = "vLLMLoRAConfig"
```

Comment:
I assume we will need different config formats for different model servers, and there will be common fields/logic that can be shared, and model server specific fields/logic. It's fine to focus on vLLM for now and refactor later though.

Reply:
Yeah maybe in refactor to extend can add an interface above and turn this into a vllm implementation of it